include/mbedtls/chacha20.h

```diff
@@ -121,6 +123,7 @@ int mbedtls_chacha20_setkey( mbedtls_chacha20_context *ctx,
  *                  messages encrypted with the same nonce and key.
  *
  * \param ctx       The ChaCha20 context to which the nonce should be bound.
+ *                  Must be initialized.
```

Comment:
Suggestion: "It must be initialized and bound to a key."

Reply:
Ok.